ToDoList/src/ToDoList.WebApi/Controllers/ToDoItemsController.cs

```diff
@@ -0,0 +1,123 @@
+namespace ToDoList.WebApi.Controllers;
+
+using Microsoft.AspNetCore.Mvc;
+using ToDoList.Domain.DTOs;
+using ToDoList.Domain.Models;
+
+[ApiController]
+[Route("api/[controller]")]
+public class ToDoItemsController : ControllerBase
+{
+    private static readonly List<ToDoItem> Items = [];
+
+    [HttpPost]
+    public IActionResult Create(ToDoItemCreateRequestDto request)
+    {
+        //map to Domain object as soon as possible
+        var item = request.ToDomain();
+
+        //try to create an item
+        try
+        {
+            item.ToDoItemId = Items.Count == 0 ? 1 : Items.Max(o => o.ToDoItemId) + 1;
+            Items.Add(item);
+        }
+        catch (Exception ex)
+        {
+            return Problem(ex.Message, null, StatusCodes.Status500InternalServerError); //500
+        }
+
+        //respond to client
+        return Created(); //201
+    }
+
+    [HttpGet]
+    public IActionResult Read(ToDoItemReadRequestDto request)
+    {
+        //map to Domain object as soon as possible
+        var items = request.FromDomain();
```

Comment:
čo tu robíš nie je čítanie všetkých items, ale nahradenie všetkých items tými, ktoré pošleš v request. To nie je úplne GET, vidíš? :-)